stress-ng.h

```diff
@@ -806,6 +822,8 @@ typedef unsigned long int __kernel_ulong_t;
 #define OPT_FLAGS_ABORT		 (0x00000200000000ULL)	/* --abort */
 #define OPT_FLAGS_CPU_ONLINE_ALL (0x00000400000000ULL)	/* --cpu-online-all */
 #define OPT_FLAGS_TIMESTAMP	 (0x00000800000000ULL)	/* --timestamp */
+#define OPT_FLAGS_DEADLINE_GRUB  0x80000000000000ULL   /* --sched-reclaim */
+
```

Comment:
can the OPT_FLAGS_DEADLINE_GRUB constant be the next free bit value (0x00001000000000ULL)

Reply:
Sure. will modify in v2.